docs/_sources/review-intro.rst.txt

```diff
@@ -31,28 +31,64 @@ The snippet below demonstrates how to send a document to Review::
     ) 
     print(submissions[0].id)
 
-WorkflowSubmissions will return a list of Submission objects, each with a unique identifier and job id. 
-In the background, the documents are OCR'd (when applicable) and passed to the workflow's models. Once
+WorkflowSubmissions will return a list of Submission objects, one for each submission, each with a unique 
+identifier and submission id. The submission ids are persistent, meaning that you can always use that id to
+check the status and retrieve the prediction results of that submission. 
+
+After submitting, the documents are OCR'd (except for image use cases) and passed to the workflow's models. Once
 the document has passed through the applicable models, it will be ready for manual review on the Review page 
 in the app.
 
+You can check the status of a submission given its id with the following call::
+
+    from indico.queries import GetSubmission
+
+    submission = client.call(GetSubmission(731))  # 731 = a submission id
+    print(submission.status) 
+    print(submission.input_filename)
+    print(submission.retrieved)
+    print(submission.result_file) # url for predicions if it's finished processing
+
+The possible statuses of a submission are: 
+
+PROCESSING - document/image is passing through the models 
+
+PENDING_REVIEW - predictions are made and the document/image is awaiting a human review
+
+PENDING_ADMIN_REVIEW - the initial reviewer rejected the document/image and awaiting admin reivew
+
+COMPLETE - the review process is complete and final predictions are ready
+
+FAILED - the document was rejected in admin review
+
+The 'retrieved' attribute of the submission object lets you know whether or not the submission results 
+has already been processed (you manage this with the UpdateSubmission call, see below).
+
+After a submission has finished "PROCESSING", you are able to retrieve the prediction results
```

Comment:
After a submission has been processed and is in COMPLETE, PENDING_REVIEW or PENDING_ADMIN_REVIEW status ...

Reply:
Made that change